src/index.html

```diff
@@ -12,7 +12,72 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page">
```

Comment:
The `body` tag should be indented with 2 spaces relative to the `html` tag to maintain proper nesting and readability.